src/datascience-ui/data-explorer/sliceControl.tsx

```diff
@@ -108,16 +124,12 @@ export class SliceControl extends React.Component<ISliceControlProps, ISliceCont
                     </summary>
                     <form onSubmit={this.handleSubmit} className="slice-form">
                         <div className="slice-enablement-checkbox-container">
-                            <input
-                                type="checkbox"
-                                id="slice-enablement-checkbox"
-                                className="slice-enablement-checkbox"
+                            <Checkbox
+                                styles={checkboxStyles}
```

Comment:
Figured out how to style FluentUI controls without hacks, went back and changed this to a FluentUI checkbox